api/kyverno/v2beta1/policy_exception_types.go

```diff
@@ -79,6 +79,19 @@ type PolicyExceptionSpec struct {
 	// Applicable only to policies that have validate.podSecurity subrule.
 	// +optional
 	PodSecurity []kyvernov1.PodSecurityStandard `json:"podSecurity,omitempty" yaml:"podSecurity,omitempty"`
+
+	// VerifyImages helps in matching the resources with the defined Image references and
+	// exclude them from applied policies
+	VerifyImages ExcludeImageReferences `json:"verifyImages,omitempty" yaml:"verifyImages,omitempty"`
```

Comment:
I wonder if it is better to exclude the images directly as follows:
```
  exceptions:
    - policyName: test
      ruleNames:
        - test
  match:
    any:
      - resources:
          kinds:
            - Pods
  imageReferences:
     - "ghcr.io/mariam*"
```
Instead of having `spec.verifyImages.imageReferences`. WDYT?

cc @JimBugwadia

Reply:
I too think that it should be ImageReferences only

Reply:
Let's keep it under the `verifyImage` field.